manifests/init.pp

```diff
@@ -61,7 +61,33 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
-) {
+
+  # Padoptions
+  #Optional[Hash[String[0],String[1]]] $padoptions = undef,
+  Optional[Hash[Enum['nocolors',
+  'showcontrols',
+  'showchat',
+  'showlinenumbers',
+  'usemonospacefont',
+  'username',
+  'usercolor',
+  'rtl',
+  'alwaysshowchat',
+  'chatandusers',
+  'lang'
+  ], String]] $padoptions = undef,
+) inherits ::etherpad::params {
+  #Merged values provides by user and default values
+  if $padoptions {
```

Comment:
Isn't https://github.com/puppetlabs/puppetlabs-stdlib#merge the same but more readable?